app/portainer/components/form-components/FileUpload/FileUploadField.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { fireEvent, render } from '@/react-tools/test-utils';
+
+import {FileUploadField} from "./FileUploadField";
+
+test('render should make the file button clickable and fie onChange event', async () => {
+
+    const onClick = jest.fn();
+    const { findByText } = render(
+        <FileUploadField title="test button" onChange={onClick} />
+    );
+
+    const button = await findByText('test button');
+    expect(button).toBeVisible();
+
+    const input = button.previousElementSibling;
```

Comment:
what if we move the input? IMO should have some label so you can do `findByLabel` or role?